src/ApparentHorizons/Strahlkorper.hpp

```diff
@@ -124,6 +151,17 @@ class Strahlkorper {
   DataVector strahlkorper_coefs_ = DataVector(ylm_.spectral_size(), 0.0);
 };
 
+namespace OptionTags {
+/// \ingroup OptionTagsGroup
+/// \ingroup SurfacesGroup
+/// The input file tag for a Strahlkorper.
+template <typename Frame>
+struct Strahlkorper {
```

Comment:
I suspect this won't be how we want to handle this tag in the long term, but since I'm not sure how we will want to I say we leave it for now.